kolibri/plugins/coach/assets/src/views/reports/learner-list-page.vue

```diff
@@ -3,20 +3,43 @@
   <div>
 
     <breadcrumbs/>
-    <h1>
+    <h1 v-if="!isRootLearnerPage">
       <content-icon
         :kind="pageState.contentScopeSummary.kind"
         colorstyle="text-default"
       />
       {{ pageState.contentScopeSummary.title }}
     </h1>
+    <report-subheading />
 
     <report-table>
       <thead slot="thead">
         <tr>
-          <header-cell :text="$tr('name')" align="left"/>
-          <header-cell :text="isExercisePage ? $tr('exerciseProgress') : $tr('contentProgress')"/>
-          <header-cell :text="$tr('lastActivity')" align="left"/>
+          <header-cell
+            align="left"
+            :text="$tr('name')"
+            :column="TableColumns.NAME"
+            :sortable="true"
+          />
+          <header-cell
+            v-if="!isRootLearnerPage"
+            :text="isExercisePage ? $tr('exerciseProgress') : $tr('contentProgress')"
```

Comment:
can go either way. since it fits easily on a single line, this seemed reasonable to me